### DISCUSSION TITLE
模板中可引用头文件相关

### DISCUSSION THREAD

**Discussion by SundayDream00** [arcsysu/SYsU-lang#43] [Q&A]
助教好，请问本系列编译器实验的非`llvm`相关头文件是否只有`SYsU-lang/librarian/sysu`目录下这些呢？如果想使用`queue`的话可以自己添加相关文件吗？会对代码编译运行有影响吗？

**Comment by Yorkking** [comment]
你试试看？（假如是标准库 std 里面的 queue，显然可以直接用）
